dependencies/maven/artifacts.snapshot

```diff
@@ -15,6 +15,9 @@
 @maven//:com_google_http_client_google_http_client_1_34_2
 @maven//:com_google_j2objc_j2objc_annotations_1_3
 @maven//:com_google_protobuf_protobuf_java_3_21_1
+@maven//:com_vaticle_typedb_typedb_cloud_runner_d0373508b6e97b61e6fd5799e9d3d5ae26680f60
+@maven//:com_vaticle_typedb_typedb_common_fffbf734614068078655248032f27270aeca26d1
```

Comment:
why is common a maven dependency now?

Reply:
Transitively from typedb-runner artifacts